samples/predict_image_classification_sample.py

```diff
@@ -0,0 +1,55 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START aiplatform_predict_image_classification_sample]
+from google.cloud import aiplatform
+from google.protobuf import json_format
+from google.protobuf.struct_pb2 import Value
+import base64
+
+
+def predict_image_classification_sample(filename: str, project: str, endpoint_id: str):
+    client_options = {
+        "api_endpoint": "us-central1-prediction-aiplatform.googleapis.com"
+    }
+    # Initialize client that will be used to create and send requests.
+    # This client only needs to be created once, and can be reused for multiple requests.
+    client = aiplatform.PredictionServiceClient(client_options=client_options)
+    location = "us-central1"
+    name = "projects/{project}/locations/{location}/endpoints/{endpoint}".format(
+        project=project, location=location, endpoint=endpoint_id
+    )
+    # See gs://google-cloud-aiplatform/schema/predict/params/image_classification.yaml for the format of the parameters.
+    parameters_dict = {"confidence_threshold": 0.5, "max_predictions": 5}
+    parameters = json_format.ParseDict(parameters_dict, Value())
```

Comment:
You can avoid calling `json_format.ParseDict` by creating a request object and passing it to the method.
```py
request = aiplatform.PredictRequest(endpoint=name)

request.parameters = {"key": "value"}) # struct.Value
request.instances.extend([{"e": "f"}, {"g": "h"}]) # Sequence[struct.Value]
request.instances.append({"l": "m"}]) # Sequence[struct.Value]

client.predict(request=request)
```

Note that for _lists of structs_ you can't assign directly (the marshaling will get confused).
```py
>>> request.instances = [{"e": "f"}, {"g": "h"}]
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/local/google/home/busunkim/.pyenv/versions/3.8.3/lib/python3.8/site-packages/proto/message.py", line 538, in __setattr__
    self._pb.MergeFrom(self._meta.pb(**{key: pb_value}))
TypeError: Value must be iterable
```

https://github.com/googleapis/proto-plus-python/issues/104 for more details

Reply:
Paging @dizcology for thoughts

Reply:
When I wrote the Doc AI samples, I created request objects, so I've replicated that here